common/resource.go

```diff
@@ -300,6 +300,51 @@ func WorkspaceData[T any](read func(context.Context, *T, *databricks.WorkspaceCl
 	}
 }
 
+func AccountData[T any](read func(context.Context, *T, *databricks.AccountClient) error) *schema.Resource {
```

Comment:
Replace with genericData that parameterizes which client is used by read function.